src/normalize/figure.ts

```diff
@@ -17,11 +19,14 @@ export default function normalizeFigure(editor: Editor, entry: NodeEntry): boole
     const [node, path] = entry
 
     if (Figure.isFigure(node)) {
-        // Figure can only be a child of sections, the document, and other
-        // figures.
+        // Figure can only be a child of sections, admonitions, the document,
+        // and other figures.
         const [parent, parentPath] = Editor.parent(editor, path)
-        if (!Section.isSection(parent) && !Figure.isFigure(parent)
-        && parentPath.length > 0) {
+        if (!Section.isSection(parent)
+            && !Figure.isFigure(parent)
+            && !Admonition.isAdmonition(parent)
+            && parentPath.length > 0
+        ) {
```

Comment:
```suggestion
        if (!Section.isSection(parent) && !Figure.isFigure(parent)
        && !Admonition.isAdmonition(parent) && parentPath.length > 0) {
```

I know this may be a bit confusing, but me and @PiotrKozlowski are using slightly different styles, and I believe consistency within a single repository is important.